rs/sns/governance/proto/ic_sns_governance/pb/v1/governance.proto

```diff
@@ -164,10 +164,14 @@ message Neuron {
     repeated NeuronId followees = 1;
   }
 
-  // The neuron's followees, specified as a map of proposal functions IDs to followees neuron IDs.
-  // The map's keys are represented by integers as Protobuf does not support enum keys in maps.
+  // The neuron's legacy followees (per proposal type), specified as a map of proposal functions IDs
+  // to followees neuron IDs. The map's keys are represented by integers as Protobuf does not
+  // support enum keys in maps.
   map<uint64, Followees> followees = 11;
 
+  // The neuron's followees, specified as a map of proposal topics IDs to followees neuron IDs.
+  map<int32, Followees> topic_followees = 19;
```

Comment:
I think key type can be enum?

Reply:
Unfortunately not. I did try, though.

Reply:
Ah. You are right: https://protobuf.dev/reference/protobuf/proto3-spec/#:~:text=Map%20Field